entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+# Set up ssh known hosts and agent
+ssh-keyscan -t rsa github.com >> /etc/ssh/ssh_known_hosts
+eval `ssh-agent -s`
+ssh-add - <<< "$SSH_PRIVATE_KEY"
+
+# split single parameter of this script into multiple params for the command
+eval "set -- $1"
+git-filter-repo "$@"
+
+git remote add remotetarget "git@github.com:$TARGET_ORG/$TARGET_REPO.git"
+git push remotetarget HEAD:"$TARGET_BRANCH"
```

Comment:
(Optional comment) the documentation says `git push` can also accept a URL directly:

```suggestion
git push "git@github.com:$TARGET_ORG/$TARGET_REPO.git" HEAD:"$TARGET_BRANCH"
```